tests/pom.xml

```diff
@@ -223,6 +239,54 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
               <autoCreateCustomNetworks>true</autoCreateCustomNetworks>
               <startParallel>false</startParallel>
               <images>
+                <!-- ##### Mongo DB instance for the device registry ##### -->
+                <image>
+                  <name>${docker.image.org-name}/hono-mongodb-test</name>
+                  <alias>mongo</alias>
+                  <build>
+                    <imagePullPolicy>IfNotPresent</imagePullPolicy>
```

Comment:
IMHO we should also skip building the image if the MongoDB profile is disabled

Reply:
It makes sense and I have now skipped building the image if the MongoDB profile is disabled. Would you mind taking a look again?